content/apidocs-mxsdk/apidocs/team-server-api.md

```diff
@@ -4,203 +4,411 @@ category: "API Documentation"
 menu_order: 65
 ---
 
+{{% alert type="warning" %}}
+The API description on this page refers to the new team server API implementation. You can access the previous team server API implementation [here](old-version/team-server-api.md).
```

Comment:
Let's clarify it a bit as follows (new stuff in _italic_):

You can access _the documentation for_ the previous team server API implementation ...

Reply:
I agree. It was not so clear before.